bin/emqx

```diff
@@ -652,6 +658,20 @@ if [ -z "$COOKIE" ]; then
     die "Please set node.cookie in $EMQX_ETC_DIR/emqx.conf or override from environment variable EMQX_NODE__COOKIE"
 fi
 
+## check if OTP version has mnesia_hook patch; if not, fallback to
+## using Mnesia DB backend.
+if [[ "${EMQX_DB__BACKEND:-}" != "mnesia"
+      || "${EMQX_DB__ROLE:-}" != "core" ]]; then
+  if [[ "$(get_config_value 'db.backend')" == "rlog"
```

Comment:
check IS_BOOT_COMMAND before executing get_config_value because it's quite expensive for other commands such as `ctl`